README.md

```diff
@@ -1,47 +1,92 @@
 ---
 page_type: sample
 languages:
-- csharp
+  - csharp
 products:
-- azure
-extensions:
-- services: Service-Bus
-- platforms: dotnet
+  - azure
+  - dotnet
+  - azure-service-bus
+description: "This sample shows how to manage Service Bus manage Queues with claims based authorization"
+urlFragment: service-bus-dotnet-manage-with-claims-based-authorization
 ---
 
-# Getting started on managing Service Bus with claims based authorization in C# #
+# Getting started on managing Service Bus with claims based authorization in C#
 
- Azure Service Bus basic scenario sample.
- - Create namespace with a queue and a topic
- - Create 2 subscriptions for topic using different methods.
- - Create send authorization rule for queue.
- - Create send and listener authorization rule for Topic.
- - Get the keys from authorization rule to connect to queue.
- - Send a "Hello" message to queue using Data plan sdk for Service Bus.
- - Send a "Hello" message to topic using Data plan sdk for Service Bus.
- - Delete namespace
+## Prerequisites
 
+To complete this tutorial:
 
-## Running this Sample ##
+If you don't have an Azure subscription, create a [free account] before you begin.
```

Comment:
I feel like this should be fleshed out with more context.   An example of what we use elsewhere:

```suggestion
- **Azure Subscription:**  To use Azure services, including Azure Service Bus, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
```